docs/editor/intellisense.md

```diff
@@ -0,0 +1,162 @@
+---
+Order: 4
+Area: editor
+TOCTitle: IntelliSense
+ContentId: 80f4fa1e-d4c5-42cf-8b12-4b8e88c41c3e
+PageTitle: IntelliSense in Visual Studio Code
+DateApproved: 8/15/2016
+MetaDescription: One of the core features of Visual Studio Code is IntelliSense.  Set breakpoints, step-in, inspect variables and more.
+---
+
+# IntelliSense
+
+IntelliSense is a general term for a variety of code editing features, including: code completion, parameter info, quick info, and list members. IntelliSense is described in a number of ways such as "code completion", "content assist", and "code hinting."
+
+![IntelliSense demo](images/intellisense/intellisense.gif)
+
+## Where is IntelliSense for my language? 
+
+IntelliSense is provided in VS Code for JavaScript, TypeScript, JSON, HTML, CSS, Less, and Sass out of the box. Other languages have word completions and can be configured to have IntelliSense by installing a language extension. 
+
+Below are the most popular language extensions in the [Marketplace](https://marketplace.visualstudio.com/vscode). Click on an extension tile below to read the description and reviews to decide which extension is best for you. 
+
+> **Tip:** For configuring and troubleshooting JavaScript IntelliSense see [here](/docs/languages/javascript#configuring-intellisense).
+
+<div class="marketplace-extensions-languages"></div>
+
+## IntelliSense Features
+
+If a language service knows possible completions, the IntelliSense suggestions will pop up as you type. If you continue typing characters, the list of members (variables, methods, etc.) is filtered to include only members containing your typed characters. Pressing `kbstyle(Tab)` or `kbstyle(Enter)` will insert the selected member. 
```

Comment:
This is the first mention of a language service.  I think you should briefly describe what one is and how it's the language services code understanding that delivers intelligence completions.